companyweb_base/views/companyweb_partner_view.xml

```diff
@@ -0,0 +1,270 @@
+                            <field
+                                string="Status"
+                                name="cweb_companystatus"
+                                attrs="{'invisible':[('cweb_companystatus_enable', '!=', True )]}"
+                            />
+                        </group> <!--end of group name-->
+                        <group
+                            name="cweb_address"
+                            string="Address"
+                            attrs="{'invisible':[('cweb_address_enable', '!=', True )]}"
+                        >
+                            <field
+                                name="type"
+                                groups="base.group_no_one"
+                                attrs="{'invisible': [('is_company','=', True)], 'readonly': [('user_ids', '!=', [])]}"
+                            />
+                            <span class="o_form_label o_td_label" name="cweb_address">
+                                <b
+                                    attrs="{'invisible': [('type', '!=', 'contact')]}"
+                                >Seat</b>
+                            </span>
+                                <div class="o_address_format">
+                                    <field
+                                    name="cweb_street"
+                                    class="o_address_street"
+                                    attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"
+                                />
+                                    <field
+                                    name="cweb_city"
+                                    class="o_address_city"
+                                    attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"
+                                />
+                                    <field
+                                    name="cweb_zip"
+                                    class="o_address_zip"
+                                    attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"
+                                />
+                                    <field
+                                    name="cweb_country"
+                                    class="o_address_country"
+                                    attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"
+                                />
+                                </div>
+                            <span
+                                class="o_form_label o_td_label"
+                                name="cweb_show_button_address"
+                            >
+                            </span>
+                                <div>
+
+                                    <field
+                                    name="cweb_show_button_address"
+                                    invisible="1"
+                                />
```

Comment:
Wondering why the xml formatter does these weird things.

Reply:
no idea, i'll check my identation, i suppose i did a double indent at one point

Reply:
found, i idented my div after the span and it was not needed
No more issue